Makefile

```diff
@@ -1,4 +1,4 @@
-PKG := github.com/adelowo/onecache
+PKG := github.com/mikeschinkel/onecache
```

Comment:
Thanks for these changes. But you seem to have changed more than the examples. 

Can you instead leave the imports as `github.com/adelowo/onecache`. See https://splice.com/blog/contributing-open-source-git-repositories-go/ for how you can do this, basically you set up a git remote to your fork rather than change import paths.

Reply:
I forgot how GitHub's pull requests work. I was trying to figure out how to make a copy of the library so I could submit another pull request, but added an issue instead about ErrCacheMiss.

Let me try to revert those changes I make and push them.  I'll ping you when I've got those changes backed out.

Reply:
@adelowo Okay, I think I got them all backed out.  Sorry about that oversight.

Reply:
And thanks for that link. I am a 25+ year programmer but only 4 months in with GoLang so still learning...

Reply:
Everyone is actually still learning - it never stops :) 

Thanks for the PR